.vib/phpmyadmin/cypress/cypress/integration/phpmyadmin_spec.js

```diff
@@ -1,16 +1,9 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows the user to log in and log out', () => {
-  cy.login();
-  cy.contains('[role="alert"]', 'Access denied').should('not.exist'); //checks if login was successful
-  cy.get('a[title="Log out"]').click();
-  cy.contains('#login_form', 'Log in');
-});
-
 it('allows creating a database and a table', () => {
   cy.login();
-  cy.visit('index.php?route=/server/databases');
+  cy.visit('/index.php?route=/server/databases');
```

Comment:
@mdhont Could you please check the selectors used in the test we will be using? I recall there is room for improvement in most of the used selectors. This will help to reduce flakiness and better understand the code.